src/EnergyPlus/AirLoopHVACDOAS.cc

```diff
@@ -693,102 +669,76 @@ namespace AirLoopHVACDOAS {
                     } else if (SELECT_CASE_var == "HEATEXCHANGER:AIRTOAIR:SENSIBLEANDLATENT") {
```

Comment:
For HEATEXCHANGER:AIRTOAIR:FLATPLATE the function argument is "errorsFound". Why isn't this InletNodeErrFlag or OutetNodeErrFlag?